style.css

```diff
@@ -0,0 +1,69 @@
+body {
+  background-color: #fff;
+  width: 375px;
+  height: 718px;
+  background-image: url("images/mobile.png");
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.Header {
+  font-family: 'Poppins', sans-serif;
+  font-style: bold;
+  font-size: 18px;
+  line-height: 20px;
+  line-height: 74%;
+  justify-content: flex-start;
+  color: #6070ff;
+}
+
+button.menu {
+  color: #6070ff;
+  position: fixed;
+  top: 1rem;
+  right: 1rem;
+  border: none;
+  background: none;
+}
+
+h1 {
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  font-style: bold;
+  font-size: 40px;
+  line-height: 52px;
+  width: 327px;
+  height: 100px;
+  top: 114px;
+  left: 24px;
```

Comment:
- [ ] To achieve the template layout you should use padding and margin properties instead of the top, left, bottom, or right